curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/6758486d05a8fcf01a20c8b4.md

```diff
@@ -0,0 +1,113 @@
+---
+id: 6758486d05a8fcf01a20c8b4
+title: Task 105
+challengeType: 19
+dashedName: task-105
+---
+
+<!-- (Audio) Brian: It was eye-opening, especially the parts about protecting data. It made me think of how we handled security in our earlier projects. -->
+
+# --instructions--
+
+Listen to the audio and answer the question below.
+
+# --questions--
+
+## --text--
+
+What did Brian think about the cybersecurity workshop?
+
+## --answers--
+
+He thought it didn’t provide useful information.
+
+### --feedback--
+
+Brian mentions that the workshop made him think about past projects, indicating it had an impact.
+
+---
+
+He thought it made him reflect on how they handled security in earlier projects.
+
+---
+
+He thought it was unrelated to their previous work.
+
+### --feedback--
+
+Brian explicitly connects the workshop to how they handled security in earlier projects.
+
+---
+
+He thought it wasn’t worth attending.
+
+### --feedback--
+
+Brian found the workshop impactful as it made him think about their past security practices.
+
+## --video-solution--
+
+2
+
+# --explanation--
+
+The phrase `make someone think` means to prompt reflection or consideration about a particular topic or situation. It often refers to something that causes a person to evaluate or analyze an issue more deeply. Example: `The documentary made me think about how I can reduce my carbon footprint.` (this means that watching the documentary helped you think about how to pollute less)
+
+In this context, the cybersecurity workshop prompted Brian to reflect on how they managed security in their projects in the past.
```

Comment:
```suggestion
`Make` someone `think` means to prompt reflection or consideration about a particular topic or situation. It often refers to something that causes a person to evaluate or analyze an issue more deeply. For example: 

`The documentary made me think about how I can reduce my carbon footprint.` - This means that watching the documentary helped you think about how to pollute less.

In this context, the cybersecurity workshop prompted Brian to reflect on how they managed security in their projects in the past.
```

I made some changes on the backticks because I think maybe "someone" should be translated. Not sure. Pls, feel free to change it back.